### PR TITLE
checkout kinflate to gopath

### PR DESCRIPTION
**Pull request by BenTheElder**
/area jobs

### CODE REVIEW

**Comment by BenTheElder** [comment]
FYI @droot
/assign @krzyzacy

File: jobs/config.json [hunk 1]
```diff
@@ -12008,4 +12008,4 @@
       "sig-big-data"
     ]
   }
-}
+}
```
Comment (krzyzacy) [inline jobs/config.json]: editor?

Comment (BenTheElder) [reply jobs/config.json]: grr

**Pull request merged**